tests/docker-compose.yml

```diff
@@ -181,7 +181,7 @@ services:
   eclair:
     restart: unless-stopped
     stop_signal: SIGKILL
-    image: acinq/eclair:release-0.7.0
+    image: acinq/eclair:latest
```

Comment:
Upgrades Eclair to v0.8.0 - TODO: Fix that version [once it is availabe on Docker Hub](https://twitter.com/_d11n_/status/1601285801119125504).